test.sh

```diff
@@ -22,12 +22,18 @@ if [ -z "$1" ]; then
   exit 1
 fi
 
+CALCULATE_FILE="./calculate_average_$1.sh"
+if [ ! -f "$CALCULATE_FILE" ]; then
+    echo "$CALCULATE_FILE does not exist."
+    exit 1
+fi
+
 for sample in $(ls src/test/resources/samples/*.txt); do
   echo "Validating calculate_average_$1.sh -- $sample"
```

Comment:
I guess we can change this to `echo "Validating $1 -- $sample"` to make it a bit cleaner

Reply:
Sounds OK to me. I'll change it.

Reply:
I forgot to change this. I think we can merge without it, and we can change it later.